test/Autofac.Integration.AspNetCore.Multitenant.Test/TestDependencies/TestableTenantIdentificationStrategy.cs

```diff
@@ -14,15 +14,16 @@ public TestableTenantIdentificationStrategy(IHttpContextAccessor httpContextAcce
 
         public bool TryIdentifyTenant(out object tenantId)
         {
-            if (_httpContextAccessor.HttpContext?.Request.Query.TryGetValue("tenant", out var tenantValues) ??
-                false)
-            {
-                tenantId = tenantValues[0];
-                return true;
-            }
+            tenantId = null!;
 
-            tenantId = null;
-            return false;
+            if (this._httpContextAccessor.HttpContext is null)
+                return false;
+
+            if (!_httpContextAccessor.HttpContext.Request.Query.TryGetValue("tenant", out var tenantValues))
+                return false;
```

Comment:
```suggestion
            if (_httpContextAccessor.HttpContext is null)
            {
                return false;
            }

            if (!_httpContextAccessor.HttpContext.Request.Query.TryGetValue("tenant", out var tenantValues))
            {
                return false;
            }
```

We haven't updated the StyleCop settings in this project yet but I think we have "no `this`" and "always include braces" turned on.

The latest StyleCop and analyzer settings, to my knowledge, are in [the Autofac.Diagnostics.DotGraph repo](https://github.com/autofac/Autofac.Diagnostics.DotGraph/tree/develop/build) and include both the enforcement for headers and the updated rulesets. I think the .editorconfig in there may also be updated based on the stuff that was discussed in the Contributors chat.

Reply:
I removed all the `this.` and used brackets in this case. But I'd agree on doing the stylecop changes in a second PR. I will create an issue.